e2e/snapshot.go

```diff
@@ -6,8 +6,8 @@ import (
 	"strings"
 	"time"
 
-	snapapi "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
-	snapclient "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned"
+	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
+	snapshotv1 "github.com/kubernetes-csi/external-snapshotter/client/v4/clientset/versioned/typed/volumesnapshot/v1"
```

Comment:
Any specific reason to change the alias/ if not can we keep this as same.

Reply:
sure.. reverted.

Reply:
why not stick with `snapclient`?

Reply:
@nixpanic its `snapclient` version, where do you see different import name?